source/common/ssl/connection_impl.cc

```diff
@@ -15,6 +15,20 @@
 namespace Envoy {
 namespace Ssl {
 
+namespace {
+// TODO(mattklein123): Currently we don't populate local address for client connections. Nothing
+// looks at this currently, but we may want to populate this later for logging purposes.
+Network::Address::InstanceConstSharedPtr
+getNullLocalAddress(const Network::Address::Instance& address) {
```

Comment:
This is a copy of the code above, right? Can we move this into a common utility function?